docs/src/examples/components/Dialog/Content/DialogExampleFooter.shorthand.tsx

```diff
@@ -0,0 +1,33 @@
+import { Button, Dialog, Box } from '@stardust-ui/react'
+import * as React from 'react'
+
+const customStylesWithComponent = compStyles => {
+  const customStyles = {
+    display: 'flex',
+    justifyContent: 'space-between',
+    alignItems: 'baseline',
+  }
+  return { ...customStyles, ...compStyles }
+}
+
+const CustomFooter = ({ styles, actions }) => {
+  return (
+    <Box styles={customStylesWithComponent(styles())}>
+      <span>any custom text in footer</span>
+      {actions}
+    </Box>
+  )
+}
+
+const DialogExampleContent: React.FC = () => (
+  <Dialog
+    cancelButton="Cancel"
+    confirmButton="Confirm"
+    content="Are you sure you want to confirm this action?"
+    header="Action confirmation"
+    trigger={<Button content="Open a dialog" />}
+    footer={render => render({}, (C, p) => <CustomFooter {...p} />)}
+  />
+)
+
+export default DialogExampleContent
```

Comment:
```suggestion
export default DialogExampleFooter
```

Reply:
I tried to avoid styling concerts in this example:

```tsx
import { Button, Dialog, Flex } from '@stardust-ui/react'
import * as React from 'react'

const DialogExampleFooter = () => (
  <Dialog
    cancelButton="Cancel"
    confirmButton="Create"
    content="Are you sure you want to create a new project?"
    header="Project creation"
    footer={render =>
      render(null, (Component, props) => {
        const { styles, ...rest } = props

        return (
          <Flex styles={styles}>
            <Button text>Privacy notes</Button>
            <Flex.Item push>
              <Component {...rest} />
            </Flex.Item>
          </Flex>
        )
      })
    }
    trigger={<Button content="Open a dialog" />}
  />
)

export default DialogExampleFooter
```

- we just changing root of slot to `Flex` and apply styles there
- using `Flex` we archive positioning of `Button`s, it also works with RTL properly